src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentBag.cs

```diff
@@ -1102,7 +1102,7 @@ public bool MoveNext()
 
             public T Current => _current;
 
-            object IEnumerator.Current
```

Comment:
should this be TODO-ed? (it cannot return null when T cannot be null)

Reply:
See my comment in your current coreclr PR.  This is an explicit interface implementation, so the only way this can be called is via the interface... as a result, annotations on this implementation don't really matter.